rpc_jobs/params.yml

```diff
@@ -54,6 +54,18 @@
           name: PARTITION_HOST
           default: true
           description: Enable partitioning of host data disk device
+      - string:
+          name: DATA_DISK_DEVICE
+          default: "{DATA_DISK_DEVICE}"
```

Comment:
The OnMetal multi-node and LEM hardware multi-node use different data disks (sdb, and a logical volume respectively), which means we cannot set a default in the macro.  Instead, we include the macro in the job definition and then pass DATA_DISK_DEVICE through as a parameter.